website/components/io-home-pre-footer/style.module.css

```diff
@@ -77,8 +77,22 @@
   }
 
   &:nth-of-type(1) {
-    @nest .consul & {
-      color: var(--white);
+    color: var(--white);
+
+    @nest .vault & {
+      color: var(--black);
+    }
+
+    @nest .nomad & {
+      color: var(--black);
+    }
+
+    @nest .packer & {
+      color: var(--black);
+    }
+
+    @nest .waypoint & {
+      color: var(--black);
```

Comment:
This change keeps these styles in sync with our .io implementations, and included within this PR to get a Vercel preview url.